ops/terraform/services/server/server-regression/lambda-src/spice-trigger/spice-trigger.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+from datetime import datetime
+
+def lambda_handler(event, context):
+    client = boto3.client('quicksight')
+    account_id = boto3.client('sts').get_caller_identity().get('Account')
+    data_set_id = get_data_set_id(client, account_id)
+    refresh_data_set(client, account_id, data_set_id)
+
+def get_data_set_id(client, account_id):
+    data_sets = client.search_data_sets(AwsAccountId=account_id, Filters=[
+        {
+            'Operator': 'StringLike',
+            'Name': 'DATASET_NAME',
+            'Value': 'bfd-test-server-regression'
```

Comment:
looks hardcoded to test. may want to add {env} instead here in case of multiple datasets.

Reply:
@brandoncruz3 Thanks for catching that! Fixed!